plugins/dice.js

```diff
@@ -3,8 +3,21 @@ const _ = require('lodash');
 module.exports = (function(){
 
 	let bot,
-		reDice = /([1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g,
-		reMessage = /(^(\!roll )[1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g;
+		reDice = /([1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?( *(&[g|l]t;=?|=) *[0-9]*)?/g,
+		reMessage = /(^(!roll )[1-9][0-9]{0,2})(d|D)([1-9][0-9]{0,3})((\+|-|\*|\/)([0-9]{1,3}))?/g;
+
+	function checkTarget(result, quantifier, target) {
+		let text = ' `Failure!`',
+			quantifiers = {
+				'&gt;': (a,b) => a>b,
+				'&lt;': (a,b) => a<b,
+				'&gt;=': (a,b) => a>=b,
+				'&lt;=': (a,b) => a<=b,
+				'=': (a,b) => a===b
+			};
+		if (quantifiers[quantifier.toLowerCase()](parseFloat(result), parseFloat(target))) text = ' `Success!`';
+		return text;
+	}
```

Comment:
more info on parseInt() http://stackoverflow.com/questions/16880327/why-am-i-getting-weird-result-using-parseint-in-node-js-different-result-from

you can also use `Number()` in most cases instead of parseInt